src/mgmt/srv_drpc.c

```diff
@@ -747,6 +747,127 @@ ds_mgmt_drpc_pool_get_acl(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 	free_ace_list(ace_list, ace_nr);
 }
 
+void
+ds_mgmt_drpc_list_pools(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
+{
+	Mgmt__PoolListReq		*req = NULL;
+	Mgmt__PoolListResp		*resp = NULL;
+	uint8_t				*body;
+	size_t				 len;
+	struct mgmt_list_pools_one	*pools = NULL;
+	size_t				 pools_len = 0;
+	uint64_t			 npools;
+	int			 srbuflen = 32;
+	int				 sri;		/* svcrep buf index */
+	int				 pi;		/* pool index*/
+	int				 ri;		/* svcrep index */
```

Comment:
Just a suggestion but it might be better to either have longer names or break out the loops using these into helper functions. It always sets off an alarm for me if variable names need to be commented with "what" rather than "why/how."